### DISCUSSION TITLE
Browser and authentication

### DISCUSSION THREAD

**Discussion by wcb88** [Teekeks/pyTwitchAPI#279] [Q&A]
I understand why the browser is necessary for authentication. However, I'm wondering why the browser pops up on every run of the app just to notify that it is authenticated. Other frameworks I've used haven't done this. I'm afraid users will have the reaction to it like they would have to a nag window. Am I doing something wrong? Is this not intentional behavior?

Thanks.

**Comment by st1vms** [comment]
I also would like to disable auto browser opening when the auth flow is already set up.
